### ISSUE TITLE
omitzero is not allowed

### ISSUE THREAD

**Issue by sbp-bvanb** [mgechev/revive#1242]
**Describe the bug**

Since 1.24, omitzero has been introduced.

**To Reproduce**
1. I updated revive `go install github.com/mgechev/revive@latest`
2. I run it with the following flags & configuration file:

```
Something          []string `json:"something,omitzero"`
```

struct-tag: unknown option 'omitzero' in JSON tag (revive)

```shell
# flags

revive ...
```

```toml
# config file
```

**Expected behavior**

**Logs**

**Desktop (please complete the following information):**
 - OS: [e.g. Ubuntu 18.04]
 - Version of Go

**Additional context**

**Comment by chavacava** [comment]
Hi @sbp-bvanb thanks for reporting the issue. Support for `omitzero` was added by #1238. The next release will include that enhancement.

**Comment by sbp-bvanb** [comment]
@chavacava We are using golangci-lint. I did not check whether that tool uses 1.7.0 of revive under the hood.

**Comment by sbp-bvanb** [comment]
It seemed to have been [merged](https://github.com/golangci/golangci-lint/pull/5422). We will have to wait for the next release of golangci-lint.

**Comment by chavacava** [comment]
AFAIK, The last version of golangci-lint does not include v1.7.0 of revive.
